### PR TITLE
jskeus 1.0.10 (new formula)

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

File: Library/Formula/euslisp.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+class Euslisp < Formula
+  desc "Integrated programming system for the research on intelligent robots"
+  homepage "https://github.com/euslisp/EusLisp"
+  url "https://github.com/euslisp/EusLisp/archive/EusLisp-9.15.tar.gz"
+  version "9.15"
+  sha256 "74cf37606bd9fcb38ddcd6dcfbbb0989375faa9c18ada3dff6e39d610d966ea9"
+  head "https://github.com/euslisp/EusLisp.git"
+
+  depends_on :x11
```
Comment (DomT4) [inline Library/Formula/euslisp.rb]: Things with mandatory X11 dependencies should be filed at `homebrew/x11`

File: Library/Formula/jskeus.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+class Jskeus < Formula
+  desc "EusLisp software developed and used by JSK at The University of Tokyo."
+  homepage "https://github.com/euslisp/jskeus"
+  url "https://github.com/euslisp/jskeus/archive/1.0.10.tar.gz"
+  version "1.0.10"
+  sha256 "038d394438dff837b9268dbec48063b33532669d5786201e2e67433fade371f1"
+  head "https://github.com/euslisp/jskeus.git"
+
+  depends_on :x11
```
Comment (DomT4) [inline Library/Formula/jskeus.rb]: Same here re X11.

**Comment by wkentaro** [comment]
> Things with mandatory X11 dependencies should be filed at homebrew/x11

I see.

**Comment by wkentaro** [comment]
moved to https://github.com/Homebrew/homebrew-x11/pull/143.

**Pull request closed without merging**